### PR TITLE
dna-cli 0.0.1 (new formula)

### PR DESCRIPTION
**Pull request by fengleicn**
This is a cli tool for DevOps, please let me upload, it will countines developed

- [ y] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [y ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ y] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ y] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ y] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [y ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/dna-cli.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Documentation: https://docs.brew.sh/Formula-Cookbook
+#                https://rubydoc.brew.sh/Formula
+# PLEASE REMOVE ALL GENERATED COMMENTS BEFORE SUBMITTING YOUR PULL REQUEST!
```
Comment (SMillerDev) [inline Formula/dna-cli.rb:3]: ...

File: Formula/dna-cli.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+class DnaCli < Formula
+  desc ""
+  homepage ""
```
Comment (SMillerDev) [inline Formula/dna-cli.rb:6]: Needs to be filled

**Review by SMillerDev — CHANGES_REQUESTED** [review]
Please follow the instructions, this isn't even close to acceptable

**Review by carlocab — CHANGES_REQUESTED** [review]
This installs a pre-built binary, which is not acceptable in Homebrew/core. See the [docs](https://docs.brew.sh/Acceptable-Formulae#we-dont-like-binary-formulae).

Please don't mark items in the PR template as done when you did not do them. This formula certainly would not pass `brew audit --new`, for example.

**Pull request closed without merging**